install/installer/pkg/components/content-service/configmap.go

```diff
@@ -29,7 +29,7 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 
 	cscfg := config.ServiceConfig{
 		Service: baseserver.ServerConfiguration{
-			Address: fmt.Sprintf(":%d", RPCPort),
+			Address: fmt.Sprintf("0.0.0.0:%d", RPCPort),
```

Comment:
will this break anything? :) 
I kind of remember a similar PR that changed from `:` to either `0.0.0.0` or `127.0.0.1` and it broke things. 
So just want to double check here.

Reply:
From my tests, seemingly not. I previously tried `127.0.0.1` and that _DID_ break it. It looks like this is the listening address - if you have `127.0.0.1:<port>` then it only allows for things going to that address - if you use `0.0.0.0` then it accepts requests via anything (eg, `localhost`, `127.0.0.1`, `<IP`> and `<FQDN>`)